packages/frontend/core/src/modules/app-tabs-header/views/app-tabs-header.tsx

```diff
@@ -82,27 +83,68 @@ const WorkbenchTab = ({
   const activeViewIndex = workbench.activeViewIndex ?? 0;
   const onContextMenu = useAsyncCallback(
     async (viewIdx: number) => {
-      await tabsHeaderService.showContextMenu?.(workbench.id, viewIdx);
+      const action = await tabsHeaderService.showContextMenu?.(
+        workbench.id,
+        viewIdx
+      );
+      switch (action?.type) {
+        case 'open-in-split-view': {
+          track.$.appTabsHeader.tabs.openInSplitView();
```

Comment:
`$.appTabs.$.xxxTab({ action: 'pin' | 'unpin' | 'close' })` would be simpler.
and `openInSplitView` is already tracked in each entry

Reply:
im not sure. by xxxTab I guess you are referring to `tabAction` or `tabContextMenu` in this case?  I think is actually the "control" part.
in our product team's definition, the event name should reflects the actual action name tho.

Reply:
control is not required anymore, as long as the events reported can be identified unique

Reply:
isn't that control is defined by product team?

Reply:
e.g., control involves events like `mid click` and `cmd click` etc. I think using unified action name is OK, but removing control may be against operation team's design.

Reply:
that could be specified by args `xxxTab({ action: 'close', ctrl: 'midClick' })`, `xxxTab({ action: 'pinTab' })`
`pinTab` doesn't require the ctrl parameter because of it can only be triggered by tab menu, we need any other information to identity such event